docs/how-to/generate-blockchain.md

```diff
@@ -0,0 +1,35 @@
+# How to Generate Blockchain
+
+It is possible situation when blockchain is not advanced for more than `k` slots.
+In this case we should generate it. Use `cardano-block-gen` command for it.
+
+## Basic Command
+
+The simplest example of the command:
+
+```
+$ stack exec -- cardano-block-gen                \
+    --system-start 0                             \
+    --configuration-file node/configuration.yaml \
+    --configuration-key mainnet_full             \
+    --blocks 1000                                \
+    --generated-db new-db
+```
```

Comment:
This command won't work, because it's impossible to create valid blocks without secret keys. There is `--secret` parameter which can be used to pass secret key (multiple keys can be passed).
Also while I was reviewing it, I realized that current implementation of block-gen won't work for mainnet, because there we use heavyweight delegation. I created https://issues.serokell.io/issue/CSL-1712 to take care of it.
DevOps should pass secret keys to which bootstrap stakeholders delegated heavily.

Reply:
Btw, @georgeee, do DevOps have access to all secret keys? If not, we probably need to provide better tooling.